src/Esfa.Vacancy.Register.Infrastructure/Services/TrainingDetailService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Threading.Tasks;
+using Esfa.Vacancy.Register.Domain.Entities;
+using Esfa.Vacancy.Register.Domain.Interfaces;
+using SFA.DAS.Apprenticeships.Api.Client;
+
+namespace Esfa.Vacancy.Register.Infrastructure.Services
+{
+    public class TrainingDetailService : ITrainingDetailService
+    {
+        public async Task<Framework> GetFrameworkDetails(int code)
+        {
+            using (var client = new FrameworkCodeClient())
```

Comment:
You mean add app settings for each external Api URLs on our website?
In which case, are we going to connect to different end points in different environments? 
If not then should we add extra configuration points which are going to be the same in all our environments?